src/components/RowProposal.vue

```diff
@@ -16,16 +16,30 @@
       <span v-text="$d(proposal.msg.payload.start * 1e3)" />
       end
       <span v-text="$d(proposal.msg.payload.end * 1e3)" />
+      <span v-if="wallets">
+        -
+        For
```

Comment:
There is proposals with more than 2 choices, we need a solution that work for any amount of proposal choices

Reply:
we have few solutions:
- showing 2 in case of only 2 available
- showing only the highest chosen option if more than 2 available